apps/mull-ui-e2e/src/integration/US-2.7.spec.ts

```diff
@@ -2,6 +2,7 @@ import 'cypress-file-upload';
 import { geolocationStub } from '../fixtures';
 import { frameSizes } from './../fixtures/frame-sizes';
 
+/* eslint-disable cypress/no-unnecessary-waiting */
```

Comment:
We don't need this anymore as I took out the `.wait(int)` in a previous PR

Reply:
will remove